templates/shared/forms/ubuntu-server-pro-tips.html

```diff
@@ -8,19 +8,17 @@ <h2 class="p-modal__title" id="modal-title">Get Ubuntu Server pro tips</h2>
       <div class="row u-no-padding">
         <div class="col-6">
           <label for="firstName" class="mktoLabel">First name:</label>
-          <input required  id="firstName" name="FirstName" maxlength="255" type="text" class="mktoField mktoRequired" >
+          <input required  id="firstName" name="firstName" maxlength="255" type="text" >
           <label for="lastName" class="mktoLabel">Last name:</label>
-          <input required  id="lastName" name="LastName" maxlength="255" type="text" class="mktoField mktoRequired">
+          <input required  id="lastName" name="lastName" maxlength="255" type="text">
           <label for="company" class="mktoLabel">Company name:</label>
-          <input required  id="company" name="Company" maxlength="255" type="text" class="mktoField mktoRequired">
+          <input required  id="company" name="company" maxlength="255" type="text">
         </div>
         <div class="col-6">
           <label for="title" class="mktoLabel">Job title:</label>
-          <input required  id="title" name="Title" maxlength="255" type="text" class="mktoField mktoRequired">
+          <input required  id="title" name="title" maxlength="255" type="text">
           <label for="email" class="mktoLabel ">Work email:</label>
-          <input required  id="email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField mktoRequired">
-          <label for="Phone" class="mktoLabel ">Mobile/cell phone number:</label>
-          <input required id="Phone" name="Phone" maxlength="255" type="tel" class="mktoField mktoTelField  mktoRequired" required="required" />
+          <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$">
```

Comment:
Email addresses are notably tricky to validate with regular expressions. Was relying on just the `type=email` not enough?

https://stackoverflow.com/questions/201323/how-can-i-validate-an-email-address-using-a-regular-expression

Reply:
Not because the HTML validation allows emails without extensions for some reason. For example: `clem@chaumel` is valid without the pattern.